web/lib/dust-i18n.js

```diff
@@ -31,13 +21,18 @@ dust.helpers.i18n = function (chunk, context, bodies, params) {
       || (context.stack && context.stack.tail && context.stack.tail.tail && context.stack.tail.tail.head && context.stack.tail.tail.head.context && context.stack.tail.tail.head.context.locality)
       || defaultLanguage;
 
-  var translation = getTranslator(locale).translate(params.key);
-  if (params.context) {
-    translation = translation.withContext(params.context);
+  var translation = i18nHelper.getClosestTranslation(locale, params.key);
+  if (translation) {
+    if (params.context) {
+      translation = translation.withContext(params.context);
+    }
+    translation = params.count
+      ? translation.ifPlural(params.count, params.key).fetch(params.count)
+      : translation.fetch();
+  } else {
+    // console.log('Missing translation', params.key); // Uncomment this to track down missing messages.po entries used in templates
```

Comment:
Not sure I'd want to log this anyway, as I fear it would be too much noise in our logs. The intention here was for a manual way to track down missing keys. We don't have access to `server` anyway